jenkins-apb/README.md

```diff
@@ -0,0 +1,28 @@
+jenkins-apb
+======================
+
+## What it does
+This APB deploys a Jenkins server capable of managing OpenShift Pipeline builds and supporting OpenShift-based
+oauth login.
+
+## Requirements
+
+## Parameters
+* persistent: Optional, if true use a persistent volume for Jenkins data.
```

Comment:
I was doing something similar for the MySQL APB but then I reworked persistence to be part of different plans, following the idea in #68 ... the PR is WIP, so just thought I'd mention it as a discussion point: should we use plans here instead?